File: server/court_microservice/model/CommunalProblem.go
package model

type CommunalProblem struct {
	Id          string `json:"id" bson:"id"`
	Title       string `json:"title" bson:"title"`
	Description string `json:"description" bson:"description"`
	Anonymous   bool   `json:"anonymous" bson:"anonymous"`
	Address     string `json:"address" bson:"address"`
	Judgeid     string `json:"judgeid" bson:"judgeid"`
	Accepted    *bool  `json:"accepted" bson:"accepted"`
}

File: client/communal-police-client/src/app/model/communal-problem.ts
export interface CommunalProblem {
    id: string;
    title: string;
    description: string;
    imageUrl: string;
    address: string;
    reportedById: string;
    policemanId: string;
    reportId: string;
    judgeId: string;
    anonymus: boolean;
    municipality: string
    date: string;
}